sbin/mount_nullfs/mount_nullfs.8

```diff
@@ -48,6 +48,7 @@
 The
 .Nm
 utility creates a
+.Xr nullfs 5
 null layer, duplicating a sub-tree of the file system
```

Comment:
i would remove the following null:
```suggestion
layer, duplicating a sub-tree of the file system
```

Reply:
Good idea.  I approve of removing redundant words.  I've just pushed a modified version of my patch that includes your suggestion.